.github/workflows/deploy-pages.yml

```
@@ -0,0 +1,87 @@
+        run: pnpm --filter @kurone-kito/dantalion-web-demo-web run build
+
+      - if: ${{ secrets.DANTALION_GH_PAGES_DEPLOY_KEY == '' }}
+        run: |
+          echo "::error::Missing secret DANTALION_GH_PAGES_DEPLOY_KEY. Resolve issue #14 before running Deploy Pages."
+          exit 1
+
+      - uses: actions/checkout@v6
+        with:
+          path: gh-pages
+          ref: gh-pages
+          repo: kurone-kito/dantalion
```

actionlint (1.7.12)

[error] input "repo" is not defined in action "actions/checkout@v6". available inputs are "clean", "fetch-depth", "fetch-tags", "filter", "github-server-url", "lfs", "path", "persist-credentials", "ref", "repository", "set-safe-directory", "show-progress", "sparse-checkout", "sparse-checkout-cone-mode", "ssh-key", "ssh-known-hosts", "ssh-strict", "ssh-user", "submodules", "token"

(action)